rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -332,6 +332,20 @@ def test_exec(config):
                             raise TestExecError(
                                 "Command failed....Bucket is not added into reshard queue"
                             )
+                    if config.bucket_sync_status:
+                        time.sleep(30)
+                        out = utils.bucket_sync_status(bucket.name)
+                        if out is False:
+                            log.info(
+                                "Bucket sync is not caught up with source. Try bucket sync run to update bucket sync status."
+                            )
+                            time.sleep(5)
+                            out = utils.check_bucket_sync(bucket.name)
+                            if out is False:
+                                raise TestExecError(
+                                    "Bucket sync is not caught up with source even after a bucket sync run"
+                                )
```

Comment:
We are validation same condition twice... Can we add delay into utils files and remove duplicate code